public/examples/card-payment.html

```diff
@@ -86,7 +70,7 @@
             region: 'LND',
             city: 'London',
           },
-          currencyCode: 'GBP',
+          currencyCode: 'USD',
```

Comment:
currency code was GBP to have an alignment with the rest of the verification details fields